integration-libs/cdc/components/login/cdc-user-account.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { UserAccountCoreModule } from '@spartacus/user/account/core';
+import { UserAccountOccModule } from '@spartacus/user/account/occ';
+import { CDCUserAccountComponentsModule } from './cdc-user-account-component.module';
+
+@NgModule({
+  imports: [
+    UserAccountCoreModule,
+    UserAccountOccModule,
+    CDCUserAccountComponentsModule,
```

Comment:
I believe this could be replaced to just `imports: [UserAccountModule, CDCUserLoginModule]`.
But let's leave it for now. It will be refactored anyway, when we introduce installation schematics that create wrapper modules in the customer's app.

Reply:
will check if this is necessary with CMS-component mapping.